addons/mrp/models/mrp_production.py

```diff
@@ -485,11 +485,9 @@ def _workorders_create(self, bom, bom_data):
 
         for operation in bom.routing_id.operation_ids:
             # create workorder
-            rounding = bom.product_uom_id.rounding
-            cycle_number = float_round(bom_qty / operation.workcenter_id.capacity, precision_rounding=rounding)
             duration_expected = (operation.workcenter_id.time_start +
                                  operation.workcenter_id.time_stop +
-                                 cycle_number * operation.time_cycle * 100.0 / operation.workcenter_id.time_efficiency)
+                                 bom_qty * operation.time_cycle)
```

Comment:
@lap-odoo Why ? we are not considering capacity and efficiency to find the estimated duration of workorder?